### PR TITLE
Advanced Image View

### PR DESCRIPTION
**Pull request by JoshuaRogan**
@JoshuaRogan 
* Migrate image view from Fandom creator

@vforge 
* Create `utils/vignette` - a Vignette helper, add it to the styleguide with a readme file. This should replace all usages of `vignette-js` in client apps.
* Create `components/ImagePreloader` that encapsulates process of loading image
* Make sure `components/Image` is using `components/ImagePreloader` to load image and `utils/vignette` to get the low quality image
* Add support to non-vignette images to `components/Image`
* Remove `components/Vignette`, as it should be replaced by either `components/Image`or a combination of `utils/vignette` and regular `` tag.

### CODE REVIEW

File: source/components/ImageView/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+/**
+ * Create a super low resolution image that will automatically be blurred in most browsers
+ *
+ * @param vignetteUrl
+ * @returns {String}
+ */
+function getLowRes(vignetteUrl) {
+    return vignetteUrl;
+    // return Vignette.getThumbURL(vignetteUrl, {
```
Comment (JoshuaRogan) [inline source/components/ImageView/index.js]: Need to address this but everything else is ready

File: source/components/Image/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import Image from './index';
+
+function mountImage(additionalProps = {}) {
```
Comment (JoshuaRogan) [inline source/components/Image/index.spec.js]: Good idea 💯

**Comment by vforge** [comment]
I want to merge it with `Vignette` and add handling for non-Vigentte images.

**Comment by vforge** [comment]
I can :)

**Pull request merged**